From fcb31293640e8a6de5052d3519b68b8d810070c4 Mon Sep 17 00:00:00 2001
From: Anton Gerasimov <anton@advancedtelematic.com>
Date: Wed, 22 Mar 2017 17:11:10 +0100
Subject: [PATCH] Make deployment work properly with grub2

---
 run/sota_ostree.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/run/sota_ostree.sh b/run/sota_ostree.sh
index 24191d6..90f3967 100755
--- a/run/sota_ostree.sh
+++ b/run/sota_ostree.sh
@@ -49,6 +49,8 @@ rm -f /etc/ostree/remotes.d/agl-remote.conf
 
 ostree remote add --no-gpg-verify $tls_option agl-remote "$PULL_URI"
 ostree pull agl-remote $auth_header_option "$COMMIT"
-ostree admin deploy "$COMMIT" && echo -n "$COMMIT" > /var/sota_ostree/staging
+
+export OSTREE_BOOT_PARTITION="/boot"
+ostree admin deploy --karg-proc-cmdline "$COMMIT" && echo -n "$COMMIT" > /var/sota_ostree/staging
 
 sync
-- 
2.11.1

